doc/tools/docgen_cmd.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""
+Script to generate documentation for command line utilities
+"""
+import sys
+import re
+from os.path import join as pjoin
+from os import listdir
+from subprocess import Popen, PIPE, CalledProcessError
+
+# version comparison
+from distutils.version import LooseVersion as V
+
+
+def sh3(cmd):
```

Comment:
I was looking at how other projects generate documentation. This is taken from a script which I really liked. I did not bother to change the name actually. :yum:
I can change it if you suggest doing so.
https://github.com/scikit-image/scikit-image/blob/master/doc/gh-pages.py